data/migrations/versions/022aef6a2ac7_add_index_for_backfill_workder.py

```diff
@@ -0,0 +1,21 @@
+"""add index for backfill workder
+
+Revision ID: 022aef6a2ac7
+Revises: 909d725887d3
+Create Date: 2022-02-21 13:50:43.204422
+
+"""
+
+# revision identifiers, used by Alembic.
+revision = '022aef6a2ac7'
+down_revision = '909d725887d3'
+
+import sqlalchemy as sa
+
+
+def upgrade(op, tables, tester):
+    op.create_index("manifest_compressed_size_id", "manifest", ["layers_compressed_size", "id"], unique=False)
```

Comment:
Any reason for creating the index on both the compressed_size and id and not just compressed size?

Reply:
There are 2 queries 1 that has only compressed_size is null and another that does a comparison on both id and compressed_size this index and the order the columns are defined in allows this index to serve both queries.